stories/components/picker/picker.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import { html } from 'lit';
+import { withCodeEditor } from '../../../.storybook/addons/codeEditorAddon/codeAddon';
+
+export default {
+  title: 'Components / mgt-picker',
+  component: 'mgt-picker',
+  decorators: [withCodeEditor]
+};
+
+export const picker = () => html`
+  <mgt-picker resource="me/todo/lists" scopes="tasks.read, tasks.readwrite" placeholder="Select a task list" key-name="displayName"></mgt-picker>
+`;
+
+export const MaxPages = () => html`
+  <mgt-picker resource="me/messages" scopes="mail.read" placeholder="Select a message" key-name="subject" max-pages="2"></mgt-picker>
+`;
+
+export const CacheEnabled = () => html`
+  <mgt-picker resource="/groups" placeholder="Select a group" scopes="group.read.all" key-name="displayName" cache-enabled="true" cache-invalidation-period="50000"></mgt-picker>
+`;
+
+export const events = () => html`
+  <!-- Inspect to view log -->
+  <mgt-picker resource="me/messages" scopes="mail.read" placeholder="Select a message" key-name="subject" max-pages="2"></mgt-picker>
+  <script>
+    document.querySelector('mgt-picker').addEventListener('selectionChanged', e => {
+      console.log('selectedItem:', e.detail.data)
```

Comment:
This should probably be logging the e.detail property as we aren't burying the selected item down another layer